app/controllers/catalog_controller.rb

```diff
@@ -41,6 +42,50 @@ def update_user_per_cat_page
     end
   end
 
+  # this method updates item quantities using the edit_reservation_form
+  # rubocop:disable AbcSize
+  def submit_cart_updates_form
+    flash.clear
+    quantity = params[:quantity].to_i
+    id = params[:id].to_i
+    equipment_model = EquipmentModel.find(id)
+    cart.send(:edit_cart_item, equipment_model, quantity)
+    @errors = cart.validate_all # update the errors
+    respond_to do |format|
+      format.html { redirect_to new_reservation_path }
+      format.js do
+        if cart.items.empty?
+          # refreshes page to redirect to catalog
+          render inline: 'location.reload();'
```

Comment:
We talked about this a little bit on Monday, but I'm still not quite sure how to force the page to redirect to catalog when it tries to render javascript.